service/geopmdpy_test/TestSecureFiles.py

```diff
@@ -106,6 +106,19 @@ def test_default_creation(self):
         secure_make_dirs(sess_path)
         self.check_dir_perms(sess_path)
 
+    def test_creation_with_perm(self):
```

Comment:
You'll need to add this test fixture name to the Makefile.mk in this directory to have it run with `make check`